source/SpinalHDL/Libraries/utils.rst

```diff
@@ -54,6 +54,24 @@ State less utilities
      - | Apply a mask on x to only keep the bit set from ``requests``.
        | it start looking in ``requests`` from the ``ohPriority`` position.
        | For example if ``requests`` is "1001" and ``ohPriority`` is "0010", the ``roundRobin`` function will start looking in `requests` from its second bit and will return "1000".
+   * - | MuxOH(
+       |   oneHot : BitVector,
+       |   inputs : Seq[T]
+       | )
+       | MuxOH(
+       |   oneHot : collection.IndexedSeq[Bool],
+       |   inputs : Iterable[T]
+       | )
+       | MuxOH(
+       |   oneHot : BitVector,
+       |   inputs : Vec[T]
+       | )
+       | MuxOH(
+       |   oneHot : collection.IndexedSeq[Bool],
+       |   inputs : Vec[T]
+       | )
+     - T
+     - Returns the muxed ``T`` from the ``inputs`` based on the ``oneHot`` vector.
```

Comment:
Having to much line text in a table is a readability issue IMHO. I suggest to boild down the command example to just:
```
   * - | MuxOH(
       |   oneHot : IndexedSeq[Bool],
       |   inputs : Seq[T]
       | )
```

Reply:
Ok. Is there by the way to link the documentation to the code?

Reply:
yes there it is.
There should be a scaladoc api documentation hosted in the main repo (here: http://spinalhdl.github.io/SpinalHDL).
So you can do this:
```
`MuxOH <http://spinalhdl.github.io/SpinalHDL/#spinal.lib.MuxOH$>`_
```
So MuxOH will be an hiperlink to the scaladoc reference.